composer.json

```diff
@@ -100,7 +100,7 @@
     },
     "conflict": {
         "phpdocumentor/reflection-docblock": "<3.0||>=3.2.0,<3.2.2",
-        "phpdocumentor/type-resolver": "<0.2.1",
+        "phpdocumentor/type-resolver": "<0.5.0",
```

Comment:
no need to bump: instead, just skip the tests that cannot run when applicable

Reply:
The tests are just new entries in an existing dataprovider. What would be the better way to skip them ? If thought of not adding them, but they won't show as skipped.

Reply:
I'd suggest to just create a new test case with separate provide method, + skipping logic inside